citation-server/src/graphql/mutation.js

```diff
@@ -11,7 +11,8 @@ import {
 import winston from 'winston';
 
 import {writeObject} from '../gitasdb/write';
-import {PageType, ComponentType, ContentType} from './query';
+// import {buildObjects} from './query';
```

Comment:
Now that your code is working, you should remove all commented code. Git is here to keep track of old code

(This is valid for all commented code bellow, I don't repeat it each time)